lib/Service/FileService.php

```diff
@@ -123,7 +123,8 @@ public function getFileFromUri(string $uri): Node {
 	 * @throws NotFoundException
 	 */
 	public function getFileFromAbsoluteUri(string $uri): Node {
-		list(, $userId, $path) = explode('/', $uri, 3);
+		\OC::$server->getLogger()->error('debug_getFileFromAbsoluteUri: ' . $uri);
```

Comment:
This log looks out of place

Reply:
Thanks, removed the commit which was only for getting a bit of more details on why one of the litmus tests were failing on CI